src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -1,6 +1,7 @@
 # cloud.yaml contains the default configuration that's made available on the
 # internet.
 client:
+  firetweetversion: "0.0.6"
```

Comment:
Yeah, that was my thinking. I like regenerating the config more often. We could just run ``genconfig.bash`` whenever we run ``deploy.bash`` too (and still read the FT version number from file or pass it as an argument).